site/src/planck-io.md

```diff
@@ -47,7 +50,7 @@ _Protocol_
   `as-url`<br/>
   `([x])`<br/>
   Coerce argument to a `goog.Uri`.
-  
+
```

Comment:
If you can, exclude whitespace changes in this file.

Reply:
My editor is configured to automatically remove whitespace at the end of lines.

Reply:
Can you use you git client to remove spurious whitespace changes outside of changes relevant to the PR?

Reply:
Fixed in revs 0c09a32 and cdb4796.